Application/EdFi.Ods.Common/Infrastructure/Extensibility/EntityExtensionRegistrar.cs

```diff
@@ -93,6 +95,7 @@ private void RegisterAggregateExtensionEntity(Type edFiStandardEntityType, Entit
             bool isRequired)
         {
             _logger.Debug($"updating {nameof(edFiStandardEntityType)} with extension {extensionName}");
+
             if (string.IsNullOrWhiteSpace(extensionName))
             {
                 throw new ArgumentNullException($"{nameof(extensionName)} cannot be null or empty.", nameof(extensionName));
```

Comment:
This should probably thrown `ArgumentException` rather than `ArgumentNullException` because the argument may not actually be null, based on the logic.

Reply:
existing code not changing